lib/src/client/channel.dart

```diff
@@ -47,6 +47,7 @@ abstract class Channel {
       {bool passive = false,
```

Comment:
Can you please update the doc block for this method to include:

```
/// The [declare] flag can be set to false to skip the exchange declaration step
/// for clients with read-only access to the broker.
```

Reply:
Done :)